api/src/pcapi/routes/backoffice/offerers/offerer_blueprint.py

```diff
@@ -260,6 +261,81 @@ def get_stats(offerer_id: int) -> utils.BackofficeResponse:
     return render_template(
         "offerer/get/stats.html",
         stats=data,
+        offerer_id=offerer_id,
+    )
+
+
+def _get_offerer_revenues_per_year(
+    offerer_id: int,
+) -> dict[str, dict[str, float]]:  # {year: {"ind": amount, "ind": amount}
+    individual_totals_query = sa.select(sa.func.jsonb_object_agg(sa.text("year"), sa.text("sum_amount"))).select_from(
+        sa.select(
+            sa.case(
+                (
+                    bookings_models.Booking.dateUsed.is_not(None),
+                    sa.extract("year", bookings_models.Booking.dateUsed),
+                ),
+                else_=0,
+            ).label("year"),
+            sa.func.sum(bookings_models.Booking.total_amount).label("sum_amount"),
+        )
+        .filter(
+            bookings_models.Booking.offererId == offerer_id,
+            bookings_models.Booking.status != bookings_models.BookingStatus.CANCELLED.value,
+        )
+        .group_by("year")
+        .subquery()
+    )
+    individual_totals: dict[str, float] = db.session.execute(individual_totals_query).scalar() or {}
+
+    collective_totals_query = sa.select(sa.func.jsonb_object_agg(sa.text("year"), sa.text("sum_amount"))).select_from(
+        sa.select(
+            sa.case(
+                (
+                    educational_models.CollectiveBooking.dateUsed.is_not(None),
+                    sa.extract("year", educational_models.CollectiveBooking.dateUsed),
+                ),
+                else_=0,
+            ).label("year"),
+            sa.func.sum(educational_models.CollectiveStock.price).label("sum_amount"),
+        )
+        .select_from(
+            educational_models.CollectiveBooking,
+        )
+        .join(
+            educational_models.CollectiveStock,
+            onclause=educational_models.CollectiveStock.id == educational_models.CollectiveBooking.collectiveStockId,
+        )
+        .filter(
+            educational_models.CollectiveBooking.offererId == offerer_id,
+            educational_models.CollectiveBooking.status != bookings_models.BookingStatus.CANCELLED.value,
+        )
+        .group_by("year")
+        .subquery()
+    )
+    collective_totals: dict[str, float] = db.session.execute(collective_totals_query).scalar() or {}
+
+    years = set(individual_totals) | set(collective_totals)
+
+    return {
+        ("En cours" if year == "0" else year): {
+            "individual": individual_totals.get(year, 0),
+            "collective": collective_totals.get(year, 0),
```

Comment:
Détail : parce que tu annonces un float : 
```suggestion
            "individual": individual_totals.get(year, 0.0),
            "collective": collective_totals.get(year, 0.0),
```